SECURITY.md

```
@@ -0,0 +1,132 @@
+# Security Policy
+
+## Reporting Security Issues
+
+If you discover a security vulnerability, please email the maintainers directly instead of opening a public issue.
+
+## Security Measures
+
+### 1. Environment Variables
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)